packages/itwinui-css/src/color-picker/color-picker.scss

```diff
@@ -1,39 +1,38 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
-@import '../style/index';
 @import '../surface/index';
 @import '../button/borderless';
 @import '../utils/index';
 
```

Comment:
Why color picker border went from darker same color to just gray?

Reply:
this one also changed to solid color:
![image](https://user-images.githubusercontent.com/36186912/185600264-f356b0cb-fbcb-4fe2-8936-12fdd0e91a6c.png)

Reply:
I think these changes are intentional and verified

Reply:
For real??? Previously swatches looked better in my opinion.